Lib/test/test_io/__init__.py

```diff
@@ -1,3 +1,26 @@
+"""Tests for the io module and its implementations (_io and _pyio)
+
+Tests are split across multiple files inside `test_io` to increase
```

Comment:
The bullet list includes files not in the folder, so perhaps:
```suggestion
Tests are split across multiple files to increase
```